app/lib/features/profile/pages/my_profile_page.dart

```diff
@@ -235,42 +231,66 @@ class MyProfile extends ConsumerWidget {
-                            context.pushNamed(Routes.settingSessions.name),
-                        label: const Text('Sessions'),
-                      ),
-                      const SizedBox(height: 30),
-                      OutlinedButton.icon(
-                        icon: const Icon(Atlas.exit_thin),
-                        onPressed: () => logoutConfirmationDialog(context, ref),
-                        label: const Text('Logout'),
+                      const SizedBox(height: 5),
+                        Container(
+                        margin: const EdgeInsets.symmetric(vertical: 10),
+                        decoration: BoxDecoration(
+                          color: const Color(0xFF122D46),
+                          borderRadius: BorderRadius.circular(10),
+                        ),
+                        child:    Column(
+                          children: [
+                             ProfileItemTile(icon: Atlas.check_shield,title: 'Linked Devices',onPressed: () =>
+                            context.pushNamed(Routes.settingSessions.name),color: Colors.white,),
+                            const Divider(indent: 40,endIndent: 10,),
+                             ProfileItemTile(icon: Atlas.gear,title: 'Settings',onPressed: () =>
+                            context.pushNamed(Routes.settings.name),color: Colors.white,),
+                          ],
+                        ),
                       ),
-                      const SizedBox(height: 45),
-                      OutlinedButton.icon(
-                        style: OutlinedButton.styleFrom(
-                          foregroundColor: AppTheme.brandColorScheme.error,
-                          backgroundColor: AppTheme.brandColorScheme.onError,
-                          side: BorderSide(
-                            width: 1,
-                            color: AppTheme.brandColorScheme.error,
+                    const SizedBox(height: 5),
+
+                    //Not implemented yet 
+                    Visibility(
+                      visible: false,
+                      child: Container(
+                          margin: const EdgeInsets.symmetric(vertical: 10),
+                          decoration: BoxDecoration(
+                            color: const Color(0xFF122D46),
+                            borderRadius: BorderRadius.circular(10),
+                          ),
+                          child:    Column(
+                            children: [
+                               ProfileItemTile(icon: Atlas.bell_reminder,title: 'Notifications',onPressed: () =>
+                              context.pushNamed(Routes.settingSessions.name),color: Colors.white,),
+                              const Divider(indent: 40,endIndent: 10,),
+                               ProfileItemTile(icon: Icons.star_border_outlined,title: 'Rate us',onPressed: () =>
+                              context.pushNamed(Routes.settings.name),color: Colors.white,),
```

Comment:
Let's not add more dead-buttons. Either we put something useful behind this button (not just moving to `settings`) or it shouldn't appear in the UI. We are trying to remove the "buttons that don't do anything and thus appear broken to the user" from the UI...

Reply:
Notifications and Rate us buttons are not visible to the users 
![image](https://github.com/acterglobal/a3/assets/47295684/8ae8314b-d119-4153-a4b6-d750bd3a725e)

Reply:
what happened to "deactivate account"? We need that (Apple AppStore requirement)

Reply:
Regarding that , I noticed that we have logout and deactivate account in the Settings Menu , Or should I just include it in MyProfile page ?
![image](https://github.com/acterglobal/a3/assets/47295684/466c62b6-e9f1-4c18-b8e3-e2026c43ec99)

Reply:
@kokodavid ah, right. no, that's fine. my bad. keep it as is.